e2e/fixtures/caseData/mandatoryWithMaxChildren.json

```diff
@@ -1,5 +1,5 @@
 {
-  "state": "SUBMITTED",
+  "state": "Open",
```

Comment:
This change is because we want to make sure that solicitors don't have access before submission and immediately after, when dealing with respondent solicitors they get their permissions immediately after submission as they can enter the info during the open state, where as child solicitors can only be added post submission